ui/apps/everest/.e2e/release/pitr.e2e.ts

```diff
@@ -446,6 +446,99 @@ test.describe.configure({ retries: 0 });
         }
       });
 
+      test(`PITR restore to new cluster [${db} size ${size}]`, async ({ page }) => {
+
+        await test.step('Create a backup to restore from', async () => {
+          await gotoDbClusterBackups(page, clusterName);
+          await clickOnDemandBackup(page);
+          await page.getByTestId('text-input-name').fill(baseBackupName + '-new-cluster');
+          await expect(page.getByTestId('text-input-name')).not.toBeEmpty();
+          await expect(page.getByTestId('text-input-storage-location')).not.toBeEmpty();
+          await page.getByTestId('form-dialog-create').click();
+          await waitForStatus(page, baseBackupName + '-new-cluster', 'Succeeded', 240000);
+        })
+
+        await test.step('Add some data after the backup', async () => {
+          await insertTestDB(
+            clusterName,
+            namespace,
+            ['9', '10', '11'],
+            ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11']
+        );
```

Comment:
Hi @tplavcic 

I copied over this logic from the existing tests (to insert some data into the DB), but this step always fails for PG, with this error:
```sh
  1) [release] › ../release/pitr.e2e.ts:449:7 › PITR › PITR restore to new cluster [postgresql size 3] @release › Add some data after th
e backup

    Error: expect(received).toBe(expected) // Object.is equality

    - Expected  - 8
    + Received  + 8

      1
    -  2
    +   2
    -  3
    +   3
    -  4
    +   4
    -  5
    +   5
    -  6
    +   6
    -  7
    +   7
    -  8
    +   8
    -  9
    +   9
```

Can you help me figure out what's wrong? It seems to work for the other tests, but not this one. 🙈

Reply:
Got the issue. Looks like the moment you add double digit numbers, PG will add a second whitespace to align everything. And the current implementation of `insertTestDb` has only 1 white space hardcoded, so it will fail for PG if you add double digits 😅